tests/attestation_plugin/host/host.c

```diff
@@ -49,6 +49,11 @@ int main(int argc, const char* argv[])
         OE_TRACE_INFO("=== tests skipped when DCAP libraries are not found.\n");
         return SKIP_RETURN_CODE;
     }
+    else
+    {
+        // setup mocks for tests
```

Comment:
```suggestion
        // set up mocks for tests
```
fix grammar ("set up" is a verb.  "setup" is a noun)